applications/RomApplication/python_scripts/randomized_singular_value_decomposition.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np
+
+class RandomizedSingularValueDecomposition():
+
+    def __init__(self, COMPUTE_U=True, COMPUTE_V=True, RELATIVE_SVD=True, USE_RANDOMIZATION=True):
```

Comment:
please add some comments to explain how this should be used and waht the parameters mean

Reply:
done